### ISSUE TITLE
Users can't vote multiple times if Allow multiple votes is checked

### ISSUE THREAD

**Issue by justiceorjustus** [YAFNET/YAFNET#587]
**Describe the bug**
Users can't vote multiple times

**To Reproduce**
1. Create poll and check allow multiple votes
2. Try to vote for multiple votes
3. Triggers a POST which states: "You can vote for many choices. You already voted."

**Expected behavior**
User can vote for multiple options

**Screenshots**

**System (please complete the following information):**
 - YAF Version: [e.g. YAF vX.XX.XX]
 - .NET Version [e.g. 4.xxx]
 - SQL Server [e.g. SQL Server 2012]
 
**Affected browser**

* [x] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Edge

**Additional context**

**Comment by w8tcha** [comment]
Is this happening for all users (incl admins)?

Atleast in my tests i cant reproduce it

**Comment by justiceorjustus** [comment]
It is happening for admins for me. The problem seems to be that when I click to vote, the page does a refresh saying that I already voted.

**Comment by justiceorjustus** [comment]
![image](https://user-images.githubusercontent.com/27013322/91660481-369cc000-eaa4-11ea-97a8-78486c593db1.png)

**Comment by justiceorjustus** [comment]
I will say I had a couple issues with the poll registering that a user already voted. If you look at the repository I shared with you, I probably made some dumb changes to get it going.

**Comment by w8tcha** [comment]
> I will say I had a couple issues with the poll registering that a user already voted. If you look at the repository I shared with you, I probably made some dumb changes to get it going.

The problem is that your fork is far behind the yaf repository. It makes it impossible to reproduce or fix the issue. Or other issues in the future.

I really don't understand why you created a fork in the first place. All the changes you make can be done without modifying the yaf source.

**Comment by justiceorjustus** [comment]
Yeah, I understand... to be honest I wasn't 100% sure I was going to go with YAF but once I got it running... I just kept modifying it until it worked for me. I'm happy that I have a functioning forum now, but I did get myself into trouble with the fork and modifications. My plan is to keep the forum as it is and once YAF is in beta or release... then I'll pull and redo my helpers and styling. 

Just like you, I have a full-time job and a full-time side project... so I had to brute force a solution together quickly. It's going to have to be "good enough" until I complete another massive update to my site that I've planned for over the next 26 weeks.

Anyways, thanks for all your help with it. I think 3.0 is a lot closer to release than you think.
